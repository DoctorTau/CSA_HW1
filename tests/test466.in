668
-220 -7810 -9203 1562 1255 2587 7196 -4371 -518 2021 3451 -4177 -7079 8343 -3864 8237 -4623 4734 8634 4544 8778 -514 -6424 -5707 -7385 -3963 2997 -6422 3219 -8458 7878 -6212 3548 -2631 8084 4810 -4161 1119 5403 -6172 9479 -2282 -6402 -8994 -8013 1571 4913 -8836 6549 1797 -8881 479 7225 5649 -6340 8144 8370 -8773 -7772 -5983 -9291 -3256 -8786 7600 -7476 5250 3193 -3882 -1882 -7663 -7926 3564 -979 -2378 6682 -12 -7479 -5493 -7344 9239 -1976 -751 9052 2538 -6895 7157 9317 -4476 7349 -7033 -8571 -8766 1534 4348 3246 2825 3030 8011 895 1329 8379 -2938 8878 -6762 -5973 -1602 452 3966 3797 -3404 -9295 215 8378 -1551 -4941 4760 5228 -970 -6452 -4959 826 -7440 9557 -521 2098 6551 -8682 4700 1809 2051 1190 -6161 8562 7922 -2415 4079 7385 6867 1764 9409 -7247 -6330 7579 8995 4200 -2704 4235 7322 5175 -6653 3344 -8129 -7421 8216 2994 -4278 5121 6372 -986 1033 -5910 -684 3743 8276 -5979 3260 9921 -1564 8247 -1859 -7672 305 9167 -8366 4589 -8278 -5618 217 -2549 -7216 6563 -5346 5072 9239 -5961 4986 4372 -7201 9659 -8377 2256 -5734 7128 -7993 1592 -6676 2432 1079 9176 -3168 6516 -9065 5744 -354 -9994 5462 -8332 -1791 -592 -8955 108 8459 4196 1258 -7602 -2862 -2898 436 5263 -7791 1876 -4775 -1488 1122 -8500 -4337 1776 -1159 3055 1422 1064 6091 -6418 9675 4393 7298 5103 3218 4951 -9869 -9409 8095 -8383 -4118 4416 -1866 -6031 9116 1257 8196 -9461 -7035 -8678 -7753 3228 -1923 5823 -9388 6929 1329 870 -1728 7844 -4577 2917 913 -568 -3197 -6554 -5962 -3852 3767 3524 -5843 6804 7786 9110 3864 5102 -9535 -6061 -2865 5182 7206 -3905 -9637 5440 -431 9270 -7944 -4506 3298 6823 -9319 8066 -5006 7716 5438 -2385 -5562 610 8190 -8878 651 7343 -7489 8083 -1332 -6604 -6851 -31 2649 2052 7797 8673 8933 345 2079 -1906 5315 -5525 3629 9104 5101 -8193 -5792 4124 -1539 9720 5715 -9423 -6896 -5946 -5586 377 7603 8760 9796 -7513 290 -9650 3450 6623 -8135 3793 -9598 6684 6585 9884 7048 -1959 -8960 8342 -6325 -919 8226 -9166 5825 -8869 -7809 -5910 4859 -1098 5237 -3243 5275 6264 9481 1386 -6706 663 -4387 7053 -1367 209 -3846 -126 -7841 -2519 -4524 5186 -5569 8896 -3731 3276 9420 -8877 -9964 -8321 -6655 2342 -2059 -2628 -1877 6517 -2217 -4664 5861 -157 -8148 -5297 2663 6155 -5059 -2318 5185 9109 -8365 3152 5785 -9588 6191 -4145 -480 7774 2992 6010 -3643 7990 8326 6716 8536 8506 746 8786 6997 -4812 3167 3531 7748 9958 -9597 4836 -8936 7807 -9459 -3101 2152 5297 6887 -9926 -7502 -8735 -66 1911 -9334 2088 -8453 -5136 8370 -7684 -6252 826 7111 4503 -4770 -5311 5733 -4728 -4193 -8681 6665 -653 2319 -4583 -9131 -2943 -1586 -4010 -1952 -1887 -1509 -7157 5351 -3543 250 -5623 -5701 8462 3278 -1234 2393 -9673 9314 -3278 -5099 1972 -2899 -1741 -2893 7306 -1301 5580 -7350 -8204 7343 -6964 6968 8526 2304 -6889 6568 8198 -4508 3645 1981 2751 -9368 1148 -3946 -6599 3649 8608 4519 2696 442 -3390 -7179 8951 5576 1724 727 -7878 7352 8448 -8307 -4076 2890 9183 -4462 4391 5523 5633 -7793 6617 -6038 -9275 674 4034 4021 4388 3710 3575 -5575 -8818 -5030 -169 2521 -9659 5080 1309 1313 -8100 -8125 7268 -2909 -2187 -5145 4020 2303 4333 1487 -2600 1883 -8093 -7792 7710 -7790 1338 -7431 -3386 -5084 8115 6475 -3188 -6565 -3709 -5143 -7743 -9250 -8127 -4687 -7581 -862 -1041 637 5974 3496 1379 8108 -2936 -6541 7603 1118 6554 -5903 5150 -2033 -8765 -268 2926 -2026 -7166 2582 8435 -2237 -9157 5909 848 6639 -94 -6222 -5889 5819 -7895 4055 -372 2698 8142 -5780 -2092 2914 -7349 7756 -3852 8862 -6732 7116 -7314 3200 8650 6789 2985 3678 -1736 -8647 3023 7069 5414 859 9185 183 24 495 7210 7735 65 1633 9880 -8142 -647 -5628 -2656 9802 7921 8648 -5443 -3079 -6925 384 -4168 4878 4025 5367 -8109 1013 7583 -2039 