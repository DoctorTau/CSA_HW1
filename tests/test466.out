-220 -7810 -518 -3864 -514 -6424 -6422 -8458 -6212 -6172 -2282 -6402 -8994 -8836 -6340 -7772 -3256 -8786 -7476 -3882 -1882 -7926 -2378 -12 -7344 -1976 -4476 -8766 -2938 -6762 -1602 -3404 -970 -6452 -7440 -8682 -6330 -2704 -4278 -986 -5910 -684 -1564 -7672 -8366 -8278 -5618 -7216 -5346 -5734 -6676 -3168 -354 -9994 -8332 -592 -7602 -2862 -2898 -1488 -8500 -6418 -4118 -1866 -8678 -9388 -1728 -568 -6554 -5962 -3852 -7944 -4506 -5006 -5562 -8878 -1332 -6604 -1906 -5792 -6896 -5946 -5586 -9650 -9598 -8960 -9166 -5910 -1098 -6706 -3846 -126 -4524 -9964 -2628 -4664 -8148 -2318 -9588 -480 -4812 -8936 -9926 -7502 -66 -9334 -5136 -7684 -6252 -4770 -4728 -1586 -4010 -1952 -1234 -3278 -7350 -8204 -6964 -4508 -9368 -3946 -3390 -7878 -4076 -4462 -6038 -8818 -5030 -8100 -2600 -7792 -7790 -3386 -5084 -3188 -9250 -862 -2936 -268 -2026 -7166 -94 -6222 -372 -5780 -2092 -3852 -6732 -7314 -1736 -8142 -5628 -2656 -4168 